CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
 # Head
 
-
--   Deprecated: `block-no-single-line`. Use `block-opening-brace-newline-after` and `block-closing-brace-newline-before` rules with the option `"always"` instead. See [the FAQs for an example](docs/user-guide/faq.md#how-do-i-disallow-single-line-blocks).
--   Deprecated: `declaration-block-no-ignored-properties`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `declaration-block-properties-order`. Use the community [`stylelint-order`](https://github.com/hudochenkov/stylelint-order) plugin pack instead. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `media-feature-no-missing-punctuation`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `selector-no-empty`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `"all-nested"` option for `at-rule-empty-line-before`. Use the `"inside-block"` option instead.
--   Deprecated: `"blockless-group"` option for `at-rule-empty-line-before`. Use the `"blockless-after-blockless"` option instead.
--   Deprecated: `"between-comments"` option for `comment-empty-line-before`. Use the `"after-comment"` option instead.
--   Deprecated: `"stylelint-commands"` option for `comment-empty-line-before`. Use the `"stylelint-command"` option instead.
--   Deprecated: `"at-rules-without-declaration-blocks"` option for `max-nesting-depth`. Use the `"blockless-at-rules"` option instead.
+See the [release planning](docs/user-guide/release-planning.md) document for information on some of these rule deprecations.
+
+-   Deprecated: 13 rules ([#2197](https://github.com/stylelint/stylelint/pull/2197) & [#2285](https://github.com/stylelint/stylelint/pull/2285)).
```

Comment:
> I thought that is what the release planning document was for

I think many, many fewer people will read the release planning document than the changelog. I, for example, have never bothered to read a release planning document ever. Changelog is the place you look when changes have been made, and you want to know what they are, how to adapt — a release planning document sounds like the place you look when you are trying to contribute or wondering what is coming next.

> Splitting out the validation rules into their own group makes sense. Any other suggestions?

Yeah, I think every rule should have an explanation about what to do next. So:

- Rules that simply wrap a third-party library. We encourage users to create and help maintain plugins for these rules.
  - `no-browser-hacks`
  - `no-indistinguishable-colors`
  - `no-unsupported-browser-features`
- Rules that did not seem useful. If you liked these rules, please create plugins for them.
  - `root-no-standard-properties`
  - `custom-property-no-outside-root`
- `stylelint-disable-reason` did not work well. Please consider contributing to #2292 for a replacement.

I think that the above, combined with spec validation and rules that already have a note, should cover it.